src/plugins/LADTable/pluginSpec.js

```diff
@@ -0,0 +1,284 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2018, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+import LadPlugin from './plugin.js';
+import Vue from 'vue';
+import {
+    createOpenMct,
+    getMockObjects,
+    getMockTelemetry,
+    getLatestTelemetry
+} from 'testTools';
+
+let openmct,
+    ladPlugin,
+    parent,
+    child;
+
+let selectors = {};
+selectors.ladTableClass = '.c-table.c-lad-table';
```

Comment:
This seems very closely dependent on the CSS and markup. Let's discuss this in standup to see if there's a way of getting the information here without it being tied so closely to the implementation?

Reply:
Yeah that's a good point. I wasn't sure of another way to do this though, maybe we can come up with something better.

Reply:
This has now been discussed, and a few options were presented, but we didn't settle on an approach. My thinking is that since we already have a convention of using `js-` classes for targeting markup from JS, let's just continue using that convention, and extend it to tests. Test specs are, after all, JavaScript.

So, let's add some `js-` classes to the markup to identify targets for testing (as few as you can get away with).

Also, let's make these CSS selectors `const`s (and name them as such, eg. `const TABLE_BODY_CLASS = js-lad-table__body` or something.